pyxrf/view/fileio.enaml

```diff
@@ -172,12 +172,13 @@ enamldef FileView(DockItem): file_view:
                     foreground = 'blue'
 
             ScrollArea: scroller:
-                MultilineField: mf:
-                    text := guihandler.text
-                    read_only = True
-                    auto_sync_text = True
-                    
-                        
+                Container:
```

Comment:
You were right here. I Put the Multiline field inside the container and now it seems to be spread across the whole thing. Might need to do some more work to make the ScrollArea actually scroll, but I'm not sure. I do not know how to use PyXRF so I was unable to generate enough log output to trigger the ScrollArea to actually have scroll bars